settings.js

```diff
@@ -152,7 +152,7 @@ window.spinnakerSettings = {
   authEnabled: authEnabled,
   authTtl: 600000,
   gitSources: ['stash', 'github', 'bitbucket'],
-  pubsubProviders: ['google', 'kafka'],
+  pubsubProviders: ['google'], // TODO(joonlim): Add amazon once it is confirmed that amazon pub/sub works.
```

Comment:
I think there should be a corresponding change in `.halconfig/settings.js`.

Reply:
Fixed. Looks like my editor auto-linted ```halconfig/settings.js```. Is this desired or should I restore those changes?

Reply:
Leaving the linted changes makes sense to me, shouldn't matter much.